packages/test/test-service-load/src/loadTestDataStore.ts

```diff
@@ -251,10 +255,26 @@ export class LoadTestDataStoreModel {
         if (partnerBlobCount > 0) {
             this.root.on("valueChanged", (v) => {
                 if (v.key.startsWith(this.partnerBlobKeyPrefix)) {
-                    // eslint-disable-next-line @typescript-eslint/no-floating-promises
-                    this.root.get<IFluidHandle>(v.key)?.get();
+                    // eslint-disable-next-line @typescript-eslint/no-non-null-assertion
+                    this.root.get<IFluidHandle>(v.key)!.get().catch((error) => {
+                        this.logger.sendErrorEvent({
+                            eventName: "ReadBlobFailed_OnValueChanged",
+                            key: v.key,
+                        }, error);
+                    });
                 }
             });
+            for (const key of this.root.keys()) {
+                if (key.startsWith(this.partnerBlobKeyPrefix)) {
+                    // eslint-disable-next-line @typescript-eslint/no-non-null-assertion
+                     this.root.get<IFluidHandle>(key)!.get().catch((error) => {
```

Comment:
Can we reuse code here? Like maybe define a lamba and use it in both places?

```ts
const action = (key) => {
                    // eslint-disable-next-line @typescript-eslint/no-non-null-assertion
                    this.root.get<IFluidHandle>(v.key)!.get().catch((error) => {
                        this.logger.sendErrorEvent({
                            eventName: "ReadBlobFailed_OnValueChanged",
                            key: v.key,
                        }, error);
                    });
};

if (partnerBlobCount > 0) {
    this.root.on("valueChanged", (v) => action);
}
for (const key of this.root.keys()) {
    action(key);
}
```

Something like that. It's also easier to read code as it's more clear what we are trying to do